08_deploy_bmo.sh

```diff
@@ -13,8 +13,24 @@ cp -r $BMOPATH/deploy ocp/.
 sed -i 's/namespace: .*/namespace: openshift-machine-api/g' ocp/deploy/role_binding.yaml
 
 # Start deploying on the new cluster
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/service_account.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role_binding.yaml
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/operator.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+KUBECONFIG=ocp/auth/kubeconfig
+oc --as system:admin apply -f $BMOPATH/deploy/service_account.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/role.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/role_binding.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/operator.yaml
+
+# Give the operator a chance to quiesce. Ideas to check instead of sleep?
+sleep 10
```

Comment:
Why are the operator related changes above necessary?  the oc cmd changes and the sleep?

Reply:
The oc changes aren't necessary... I can revert those. Just makes the commands shorter. :) The sleep may not be necessary. I just didn't want a race of the machines not being there yet since the operator would have just started.

Reply:
Oh, the Machines aren't created by this operator, so no race worries here